src/common/Themes/Themes.vcxproj

```diff
@@ -31,10 +31,14 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClInclude Include="icon_helpers.h" />
+    <ClInclude Include="ThemeListener.h" />
+    <ClInclude Include="theme_helpers.h" />
     <ClInclude Include="windows_colors.h" />
   </ItemGroup>
   <ItemGroup>
     <ClCompile Include="icon_helpers.cpp" />
+    <ClCompile Include="ThemeListener.cpp" />
```

Comment:
Why different naming styles ? :)

Reply:
I created a class, while the others were loose c++ helper files, and this library had that convention.
Should I change the class naming style, or keep the class naming style, but change it's file name?

I'm new to c++, so I'm not really sure what the convention is 😅

Reply:
coding style is ok. Just file naming should be uniform, i.e. rename `ThemeListener.h|cpp` -> `theme_listener.h|cpp`